spec/orm/activerecord_spec.rb

```diff
@@ -1195,6 +1197,14 @@ def monkey
     end
 
     describe "#remote_images_urls=" do
+      before do
+        sham_rack_app = ShamRack.at('www.example.com').stub
+        sham_rack_app.register_resource('/test.jpg', File.read(file_path('test.jpg')), 'images/jpg')
```

Comment:
@thomasfedb I started extracting a support module for ShamRack but felt this wasn't the best and simplest way to go (espacially when looking at other ShamRack stubs in `download_spec` for example).
I suggest replacing ShamRack by WebMock which provides imho a better API for stubbing external HTTP requests and it's widely used.
I replaced ShamRack by WebMock at https://github.com/mehlah/carrierwave/commit/c28633e13eb83bc49c544bf9835966fbb5aef048 to simplify/clarify some specs examples.
I'd be happy to PR this if you want to discuss it.